monkey/infection_monkey/monkey.py

```diff
@@ -8,6 +8,7 @@
 from ipaddress import IPv4Interface
 from itertools import chain
 from pathlib import Path, WindowsPath
+from socket import gethostname
```

Comment:
Let's wrap `gethostname()` in a new function: `utils.environment.get_hostname()`.